index.js

```diff
@@ -29,20 +29,15 @@ function getJSON(obj) {
 		} else {
 			obj.headers = {'User-Agent': FAKE_USER_AGENT};
 		}
-		request({
-			url: obj.url,
-			headers: obj.headers,
-			rejectUnauthorized: false
-		}, function (err, req, body) {
-			if (err) {
-				return reject(err);
-			}
+		got(obj.url, {
+			headers: obj.headers
+		}).then(request => {
```

Comment:
Can you set `rejectUnauthorized` to `false` again here?
We took this property from `node-spotify-web-helper` and I guess there was a reason. I just want to make sure to not break stuff with this PR.

`got` should support this as well: https://github.com/sindresorhus/got/search?utf8=✓&q=rejectUnauthorized&type=